Source/WebCore/page/EventHandler.cpp

```diff
@@ -1459,7 +1459,7 @@ std::optional<Cursor> EventHandler::selectCursor(const HitTestResult& result, bo
         return std::nullopt;
 
     auto renderer = node->renderer();
-    auto* style = renderer ? &renderer->style() : nullptr;
+    auto* style = renderer ? &renderer->style() : node->computedStyle();
```

Comment:
auto* style = node->computedStyle();

Also worth a comment describing why the computed style is used and not the renderStyle like it usually is the case.

Reply:
Can we always use the computed style here, not only when the node has no renderer?

Reply:
I think so, @anttijk wdyt?

Reply:
He DM me that it "should be ok"